nix/default.nix

```diff
@@ -15,5 +15,7 @@ in
       allowUnfree = true;
       android_sdk.accept_license = true;
     };
-    overlays = [ (import ./all-packages.nix) ];
+    overlays = [
+      (import ./all-packages.nix)
+    ];
```

Comment:
Was this change intentional?  I don't object to it, but I don't see the need either.